providers/deepl/lib/index.js

```diff
@@ -35,24 +35,41 @@ module.exports = {
     const rateLimitedTranslate = limiter.wrap(client.translateText.bind(client))
 
     return {
-      async translate({ text, priority, sourceLocale, targetLocale }) {
+      /**
+       * @param {{
+       *  text:string|string[],
+       *  sourceLocale: string,
+       *  targetLocale: string,
+       *  priority: number,
+       *  format?: 'plain'|'markdown'|'html'
+       * }} options all translate options
+       * @returns {string[]} the input text(s) translated
+       */
+      async translate({ text, priority, sourceLocale, targetLocale, format }) {
         if (!text) {
           return []
         }
         if (!sourceLocale | !targetLocale) {
```

Comment:
is there a reason this is a bitwise OR? Should work the same, just a bit unusual to see 😄

Reply:
Just a typo